src/Nethermind/Nethermind.Merge.AuRa/AuRaMergeBlockProcessor.cs

```diff
@@ -63,12 +60,10 @@ public class AuRaMergeBlockProcessor : AuRaBlockProcessor
             gasLimitOverride,
             contractRewriter
         )
-    {
-        _poSSwitcher = poSSwitcher;
-    }
+    { }
 
     protected override TxReceipt[] ProcessBlock(Block block, IBlockTracer blockTracer, ProcessingOptions options) =>
-        _poSSwitcher.IsPostMerge(block.Header)
+        block.IsPostMerge
```

Comment:
This is probably enough here? @MarekM25

Reply:
Check `MergePlugin.Init` method, there is a preprocessor steps that sets this property using the `PoSSwitcher`

Reply:
This change is already running in a shadowfork, and it is working just fine